pallets/file-system/Cargo.toml

```diff
@@ -16,6 +16,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec = { workspace = true }
+log = { workspace = true }
```

Comment:
Is this needed?

Reply:
I throw a warning in the file system pallet in the compute threshold function.

Reply:
But wouldn't that not be included when compiled to WASM? The runtime API is executed by instantiating a WASM blob.

Reply:
It was for when we execute tests.

Reply:
I can understand that, was not a big deal to me since I looked around and found log prints in substrate pallets as well. I can remove it if you insist.

Reply:
I really think this is nice to have though.

https://github.com/Moonsong-Labs/storage-hub/blob/f80e30150bd845de84a95e4aced971a9e363f7b0/pallets/file-system/src/utils.rs#L1206-L1212

Reply:
It's fine, leave it then if you find it useful